cmd/cassandra/main.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"strings"
+	"syscall"
+
+	kitprometheus "github.com/go-kit/kit/metrics/prometheus"
+	"github.com/gocql/gocql"
+	"github.com/mainflux/mainflux"
+	log "github.com/mainflux/mainflux/logger"
+	"github.com/mainflux/mainflux/writers"
+	"github.com/mainflux/mainflux/writers/cassandra"
+	nats "github.com/nats-io/go-nats"
+	stdprometheus "github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	sep = ","
+
+	defNatsURL  = nats.DefaultURL
+	defPort     = "8180"
+	defCluster  = "127.0.0.1"
```

Comment:
If this is cluster address, I suggest name `clusterAddr`. Also, replace IP with `localhost` if possible.

Reply:
This is not single address, but array of addresses that are part of cluster.